### DISCUSSION TITLE
SvelteKit conditional deployments with Tauri

### DISCUSSION THREAD

**Discussion by vhscom** [vercel/vercel#7459] [Help]
I developed an app with SvelteKit and `adapter-auto` and it deployed to Vercel without any additional configuration using the `vercel` command. App runs completely offline. Yesterday I added Tauri to the app to gain the benefit of a native wrapper around the app and configured `svelte.config.js` to use `adapter-static` when a native build is triggered.

Commit which introduced the deployment issue:
https://codeberg.org/vhs/oathqr/commit/37b48375ba866f61343a165a264efba6ca6d0781

Locally both native and web build targets continue to work as expected. However, when I attempted to deploy to Vercel using the `vercel` command it seems a lot more was happening than I expected as the upload size increased significantly and then I hit the following error:

```term
Vercel CLI 24.0.0
> Upload [===========---------] 53% 84.9sError! SHA1 sum of the file (dc711146223ca20c7d1e815709232ec68f5f3279) doesn't match the provided SHA1 sum (c9bb72896cc0ff2ecf060f801cd22a9051fe75b1)
Error! An unexpected error occurred!
DeploymentError: Too many requests - try again in 18 hours (more than 5000, code: "api-upload-free").
    at uploadList.<computed>.async_retry_1.default.retries (/usr/local/lib/node_modules/vercel/dist/index.js:211549:27)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

The only obvious change which could have caused this was the introduction of a `src-tauri` directory within my app which contains a nested `.gitignore` file. So my question is, are nested `.gitignore` files respected when deploying SvelteKit apps with the Vercel CLI and where can I find more information about Vercel configuration (such as the JSON file) as the CLI `--help` docs are still a bit sparse.

Thank you!

**Comment by styfle** [comment]
Try creating a `.vercelignore` https://vercel.com/guides/prevent-uploading-sourcepaths-with-vercelignore

Alternatively, deploy using using Git which will automatically utilize your `.gitignore` https://vercel.com/docs/concepts/git
